packages/button/docs/Button.stickersheet.stories.tsx

```diff
@@ -0,0 +1,247 @@
+import React from "react"
```

Comment:
Do these Button story updates need to be here?

Reply:
I think it's better that they go into their own category rather than being hidden by config, we still want to be able to access them.